src/app/backend/resource/pod/list.go

```diff
@@ -123,6 +126,7 @@ func ToPodList(pods []v1.Pod, events []v1.Event, nonCriticalErrors []error, dsQu
 		warnings := event.GetPodsEventWarnings(events, []v1.Pod{pod})
 		podDetail := ToPod(&pod, metrics, warnings)
 		podDetail.Warnings = warnings
+		podDetail.NodeName = pod.Spec.NodeName
```

Comment:
For such a small change it's not worth to create another PR. It can be done in this one.